.claude/skills/PrePR/SKILL.md

```
@@ -0,0 +1,15 @@
+---
+name: pre-pr
+description: prの下書きを作成してもらう
+---
+
+今現在のブランチで行った作業、コミットなどを元にプルリクの下書きを新規にpre_pr.mdとしてルートディレクトリに出力して
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)